components/vf-banner/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.1.0",
+  "version": "1.2.0",
```

Comment:
Are the package.json files accidental commits? They might mess with lerna?

Reply:
I seemed to have published it to NPM before it got merged 

https://www.npmjs.com/package/@visual-framework/vf-banner